package.json

```diff
@@ -5,14 +5,14 @@
     "start": "echo 'See README.md#getting-started \\n\\n'",
     "demo": "npx npm-run-all --print-label --parallel start:all open:all",
     "start:all": "npx npm-run-all --print-label --parallel start:services start:interfaces",
-    "start:services": "cd services/ && docker-compose -f  docker-compose.yml -f  docker-compose.development.yml up -d --build",
+    "start:services": "cd services/ && docker-compose -f docker-compose.yml -f docker-compose.development.yml up -d --build",
     "start:interfaces": "npx npm-run-all --print-label --parallel start:pa start:aw start:ho start:referral",
     "start:pa": "npm start --prefix interfaces/PA-App/ -- --port 8008",
     "start:aw": "npm start --prefix interfaces/AW-App/ -- --port 8080",
     "start:ho": "npm start --prefix interfaces/HO-Portal/ -- --port 8888",
     "start:referral": "npm start --prefix interfaces/Referral-App/ -- --port 8800",
     "install:all": "npx npm-run-all --print-label install:services install:interfaces",
-    "install:services": "cd services/ && docker-compose up -d --build --force-recreate",
+    "install:services": "cd services/ && docker-compose build",
```

Comment:
@elwinschmitz shouldn't this be `docker-compose build --no-cache` instead?

For reference, [up](https://docs.docker.com/compose/reference/up/) and [build](https://docs.docker.com/compose/reference/build/).

Reply:
Yeah, I left it out so you could apply the flag manually;
Like:
```
npm run install:services -- --no-cache
```

Other flags can be applied to all these commands in the same way.